scripts/uboot_helper

```diff
@@ -100,7 +100,8 @@ devices = \
       },
       'orangepi-3': {
         'dtb': 'sun50i-h6-orangepi-3.dtb',
-        'config': 'orangepi_3_defconfig'
+        'config': 'orangepi_3_defconfig',
+        'crust_config': 'orangepi_3_defconfig',
       },
```

Comment:
Please add other configs, at least those supported by Crust. If it works here, I'm pretty sure it will work with others too.

Reply:
Added it for the boards where crust ships a defconfig